### PR TITLE
R4R: Update mint doc

### PR DESCRIPTION
**Pull request by SegueII**

### CODE REVIEW

File: docs/features/mint.md [hunk 1]
```diff
@@ -35,46 +35,28 @@ Suppose `blockCostTime` is 5000 millisecond, and `inflationRate` is `4%`, then t
...
-
-    You can use `--trust-node` flag to indicate whether the connected full node is trustable or not. If you can't access to a trustable node, this command line is very helpful.
-
-    ```bash
-    iriscli bank token-stats --trust-node=false --chain-id=irishub --node=<iris-node-url>
-    ```
-
-    Example Output:
+```bash
+iris q stake pool --node=<iris-node-url>
```
Comment (zhiqiang-bianjie) [inline docs/features/mint.md:45]: `iris query staking pool`

File: docs/features/mint.md [hunk 2]
```diff
@@ -35,46 +35,28 @@ Suppose `blockCostTime` is 5000 millisecond, and `inflationRate` is `4%`, then t
...
+Example Output:
 
-    ```
+```bash
+Pool
+Loose Tokens: 1846663.900384156921391687
+Bonded Tokens: 425182.329615843078608313
+Token Supply: 2271846.230000000000000000
+Bonded Ratio: 0.187152776500000000
+```
```
Comment (zhiqiang-bianjie) [inline docs/features/mint.md:56]:
```text
bonded_tokens: "100000000000000000000"
not_bonded_tokens: "0"
```

File: docs/features/mint.md [hunk 3]
```diff
@@ -35,46 +35,28 @@ Suppose `blockCostTime` is 5000 millisecond, and `inflationRate` is `4%`, then t
...
 
-3. `/stake/pool` and `/bank/token-stats`
+`/stake/pool`
```
Comment (zhiqiang-bianjie) [inline docs/features/mint.md:58]: `/staking/pool`

File: docs/zh/features/mint.md [hunk 4]
```diff
@@ -2,79 +2,62 @@
...
-      Burned Tokens:  177.59638iris
-      Bonded Tokens:  425182.329615843078608313iris
-    ```
+```bash
+Pool
+Loose Tokens: 1846663.900384156921391687
+Bonded Tokens: 425182.329615843078608313
+Token Supply: 2271846.230000000000000000
+Bonded Ratio: 0.187152776500000000
+```
```
Comment (zhiqiang-bianjie) [inline docs/zh/features/mint.md:57]: same as above

File: docs/zh/features/mint.md [hunk 5]
```diff
@@ -2,79 +2,62 @@
...
 
-3. `/stake/pool`和`/bank/token-stats`
+**`/stake/pool`**
```
Comment (zhiqiang-bianjie) [inline docs/zh/features/mint.md:59]: staking

**Review by chengwenxi — APPROVED** [review]

**Review by zhiqiang-bianjie — APPROVED** [review]

**Pull request merged**